easybuild/easyconfigs/m/Mono/Mono-6.12.0.122-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,53 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'ConfigureMake'
+
+name = 'Mono'
+version = '6.12.0.122'
```

Comment:
Update to 6.12.0.199

Reply:
version 6.12.0.199 is a preview for Mono ?
https://download.mono-project.com/sources/mono/

Reply:
huh
https://www.mono-project.com/download/stable/#download-lin
states
> The latest Stable Mono release is: 6.12.0 Stable (6.12.0.199)

I guess they wrote it wrong. Both on docker and in the download 6.12.0.182 looks to be what should be considered stable i guess.